### ISSUE TITLE
Keyboard doesn't have any effect on gallery when it is opened using mouse.

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#224]
Windows Xampp FF, Opera and IE10
1. Right-click on image to see contenxt menu and select view option to see the gallery
2. Start pressing arrows or escape
Result: they have no effect on the gallery because focus is left on the images behind the gallery.

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
